infrastructure/templates/infrastructure/powerplant_detail.html

```diff
@@ -0,0 +1,160 @@
+{% extends "base.html" %}{% load staticfiles %}{% load humanize %}{% load date_extras %}
+
+{% block extrahead %}
+{% include "_html_head.html" with title=object.alternate_name description=object.description_rendered %}
+<style>
+    #map {
+        width:  100%;
+        height: 400px;
+        display: block;
+    }
+</style>
+<script src='{% static "js/projectmap.js" %}'></script>
+{% endblock %}
+
+{% block pagetitle %}{{ object.name }} | {{ block.super }}{% endblock %}
+
+{% block bodyclass %}powerplant{% endblock %}
+
+{% block content %}
+<div>
+    <ul class="breadcrumbs">
+        <li>
+            <a href="{% url 'database-home' %}">Database</a>
+        </li>
+        <li>
+            {{ object.name }}
+        </li>
+    </ul>
+</div>
+<article>
+    <header
+    class='decorated infrastructure{% if object.infrastructure_type %} {{object.infrastructure_type.slug}}{% endif %}'
+    >
+        <h1>{% if object.alternate_name %}{{ object.alternate_name }}
+        {% else %}{{ object.name }}{% endif %}
+        </h1>
+    </header>
+{% if object.geo %}
+    <section class="map-container">
+        <div id="map"></div>
+    </section>
+{% endif %}
+    <section class="details">
+        {% if object.description_rendered %}
+        <section class="description">
+            <h3>Description</h3>
+        {{ object.description_rendered|safe }}
+        </section>
+        {% endif %}
+
+        <section>
+          <span>Associated Projects: </span>{% for project in object.project_set.published %}<span><a href="{{ project.get_absolute_url }}">{{ project.name }}</a></span>{% endfor %}
```

Comment:
Probably should add a comma here between projects. And bold the "Associated Projects" label.